cmd/cluster/instance_deploy.go

```diff
@@ -31,8 +31,8 @@ import (
 
 var deployCmd = &cobra.Command{
 	Use:   "deploy",
-	Short: "Deploy an Astarte Instance in the current Kubernetes Cluster",
-	Long: `Deploy an Astarte Instance in the current Kubernetes Cluster. This will adhere to the same current-context
+	Short: "Deploy a minimal Astarte Instance in the current Kubernetes Cluster",
+	Long: `Deploy a minimal Astarte Instance in the current Kubernetes Cluster. This will adhere to the same current-context
```

Comment:
Maybe let's underline further that this should only be used for testing purposes